hpy/universal/src/hpymodule.c

```diff
@@ -105,6 +106,38 @@ static PyObject *do_load(PyObject *name_unicode, PyObject *path, int debug)
         goto error;
     }
 
+    char minor_version_symbol_name[258];
```

Comment:
This is the place where we load and check the ABI versions. We do not check the ABI tag here. We can get the ABI tag from the filename and use that instead or alongside of the symbols that this PR adds.